tests/test_tifs.py

```diff
@@ -5,6 +5,7 @@
 
 import hyp3_sdk.util
 import pytest
+import rioxarray
```

Comment:
```suggestion
import rioxarray  # noqa: F401
```
You can disable Flake8 for this line

https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html